django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -148,10 +148,33 @@
     border-bottom: none;
 }
 
-#changelist-filter h3 {
+#changelist-filter details summary {
+    cursor: pointer;
     font-weight: 400;
     padding: 0 15px;
-    margin-bottom: 10px;
+    margin: .6em 0 10px 0;
+    color: var(--body-quiet-color);
+}
+
+#changelist-filter details summary > * {
+    display: inline;
+}
+
+details > summary {
+    list-style-type: none;
+}
+
+details > summary::-webkit-details-marker {
+    display: none;
+}
+
+details > summary::before {
+    font-family: "Bitstream Vera Sans Mono", Monaco, "Courier New", Courier, monospace;
```

Comment:
I had to re-fine the font-family because the inherit from the body breaks it:

### Before:
![Screenshot from 2022-02-19 18-45-43](https://user-images.githubusercontent.com/16822952/154860362-53c65f3b-324d-4bdf-9fc5-7f504b45111c.png)

### After:
![Screenshot from 2022-02-19 18-47-39](https://user-images.githubusercontent.com/16822952/154860377-b5f4c437-5ec2-448f-93c5-d67a0ec26149.png)